ios/App/App/plugins/AbsAudioPlayer.swift

```diff
@@ -95,13 +99,69 @@ public class AbsAudioPlayer: CAPPlugin {
     }
     
     @objc func sendMetadata() {
-        self.notifyListeners("onPlayingUpdate", data: [ "value": !PlayerHandler.paused() ])
+        self.notifyListeners("onPlayingUpdate", data: [ "value": !PlayerHandler.paused ])
```

Comment:
I get an error here without (). I see you removed the (), is this a version difference?

Reply:
I see now you made them all getters & setters but my xcode doesn't recognize that. Maybe I need to update xcode or something

Reply:
> I get an error here without (). I see you removed the (), is this a version difference?

Sorry, I don't know how I missed that. It has to be `.paused`

> I see now you made them all getters & setters but my xcode doesn't recognize that. Maybe I need to update xcode or something

That is really weird. I am on macOS `12.3.1 (21E258)` and Xcode `13.3.1 (13E500a)`

Reply:
It is working today, not sure what happened.